tests/testthat/test-content.R

```diff
@@ -168,6 +168,12 @@ without_internet({
       con$content(name = "A name for content"),
       "https://connect.example/__api__/v1/content?name=A%20name%20for%20content&include=tags%2Cowner"
     )
+
+    uuid <- uuid::UUIDgenerate()
```

Comment:
This is fine, but FTR I don't think you need an actual guid here, you could pass `guid = 1` and assert that you `GET https://connect.example/__api__/v1/content/1?include=tags,owner`